assets/js/googlesitekit/datastore/user/user-info.test.js

```diff
@@ -38,6 +38,7 @@ describe( 'core/user userInfo', () => {
 			picture: 'https://path/to/image',
 		},
 		verified: true,
+		userInputState: 'complete',
```

Comment:
Nitpicking, but I think it would be more accurate to use an actually supported value, i.e. `completed`.